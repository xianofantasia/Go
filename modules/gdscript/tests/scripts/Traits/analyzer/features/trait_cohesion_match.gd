uses SomeTrait, OtherTrait

trait SomeTrait:
		func func_one():
				print("function one")

trait OtherTrait:
		func func_two():
				print("function two")

func test():
		func_one()
		func_two()
		print("ok")
