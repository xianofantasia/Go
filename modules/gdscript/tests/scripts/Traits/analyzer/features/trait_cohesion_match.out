GDTEST_OK
function one
function two
ok
